app/scripts/modules/docker/src/pipeline/trigger/dockerTriggerOptions.directive.js

```diff
@@ -67,6 +67,15 @@ module.exports = angular
 
     this.updateSelectedTag = (item) => {
       this.command.extraFields.tag = item;
+
+      const imageName = this.command.trigger.registry + '/' + this.command.trigger.repository;
+      const imageReference = imageName + ':' + item;
```

Comment:
Any chance `item` is a digest?

Reply:
No, as far as I know. As the function name implies, the `item` here is just the selected (Docker image) tag. I'd be happy to rename the parameter to `tag` if my understanding is correct.

Reply:
And the Docker image provider does not show digests at all. Only tags are available.

Reply:
Done, added some checks and made `registry` optional.